### PR TITLE
📅 Add last updated to resource detail page

### LINKED ISSUE #465: Add last updated indicator to resource detail
We want to let users know the last time a resource was updated. We should display this somewhere on the resource detail page. The field should already be surfaced by our `getResourceById` call, so we just need the `lastUpdated` field. Additionally, we'll likely have to convert the timestamp to something more readable.

### PR DESCRIPTION
**Pull request by Feeshay**
## Status:
:rocket: Ready

## Description

* Fixes resource detail page not loading (part of #471) - TIL about destructuring
* Add last updated to desktop and mobile resource detail pages (need to translate static text)

Resolves #465 

## Screenshots

Desktop:
image

Translated date (still need to translate static text):
image

Mobile:
image

### CODE REVIEW

File: client/src/components/desktop/ResourceDetail.jsx [hunk 1]
```diff
@@ -84,48 +85,51 @@ function ResourceDetail(props) {
     async function didMount() {
       const response = await getResourceByID(match.params.id, true);
       if (response !== null) {
-        const { res } = response;
```
Comment (jackieo5023) [inline client/src/components/desktop/ResourceDetail.jsx:87]: thanks for changing all this back 😂

File: client/src/components/desktop/ResourceDetail.jsx [hunk 2]
```diff
@@ -390,6 +394,17 @@ function ResourceDetail(props) {
               id: `resource-eligibilityRequirements-${match.params.id}`,
               defaultMessage: eligibility,
             })}`}
+          {lastUpdated && (
+            <t style={{ color: 'gray' }}>
+              {`\n\n${intl.formatMessage(
+                detailMessages.lastUpdated,
+              )} ${intl.formatDate(lastUpdated, {
```
Comment (jackieo5023) [inline client/src/components/desktop/ResourceDetail.jsx:401]: thanks for doing the format date!

File: client/src/components/desktop/ResourceDetail.jsx [hunk 3]
```diff
@@ -390,6 +394,17 @@ function ResourceDetail(props) {
               id: `resource-eligibilityRequirements-${match.params.id}`,
               defaultMessage: eligibility,
             })}`}
+          {lastUpdated && (
+            <t style={{ color: 'gray' }}>
```
Comment (jackieo5023) [inline client/src/components/desktop/ResourceDetail.jsx:398]: what is a t tag? is there a reason we can't do span/div?

**Review by jackieo5023 — APPROVED** [review]
Thanks for fixing the resource detail, generally cleaning up some code, and adding the indicator! Looks great. Just one question on what the `t` tag is though

File: client/src/components/desktop/ResourceDetail.jsx (hunk 3 above)
Comment (Feeshay) [reply client/src/components/desktop/ResourceDetail.jsx:398]: bruh idk I swear it was used for text but that might've been a completely different project lol

**Pull request merged**